gui_qt/check_failures_report.py

```
@@ -517,12 +517,12 @@ def build_check_issues_report(
 
     if not report_path and report_text is None:
         message = (
-            "任务清单中没有记录检查报告路径，也无法从翻译包目录推断。"
+            "任务记录中没有检查报告路径，也无法从翻译包目录推断。"
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)